projects/interlinked_intricate_network/index.md

```diff
@@ -0,0 +1,218 @@
+--- 
+authors: 
+  - "Jakob von Kietzell"
+authors-url: 
+  - "https://github.com/jakobvk"
+layout: project
+title: Interlinked Intricate Network
+type: project
+---
+
+# Interlinked Intricate Network
```

Comment:
Please start at h2 the h2 will be added by the system.  See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax